blackbox/sqllogictest/src/tests.py

```diff
@@ -59,7 +59,7 @@ def test_suite():
         port=CRATE_HTTP_PORT,
         transport_port=CRATE_TRANSPORT_PORT,
         settings={
-            'stats.enabled': True,
+            'stats.enabled': 'true',
```

Comment:
because es complains that this is not a valid boolean value when tries to parse it from string